storyblok/storyblok-js-client
# fix(int-663): adding check resolve relations

- If V1 api call is forced from the sdk, an error will trigger without this check.



## Pull request type

Jira Link: [INT-663](https://storyblok.atlassian.net/browse/INT-663)




## How to test this PR
Try to make the API call to resolve relations, forcing to V1. None should be return, neither trigger any error.


## What is the new behavior?



-
-
-

## Other information
